src/WpfMath/Atoms/RowAtom.cs

```diff
@@ -67,7 +65,7 @@ private RowAtom(SourceSpan source, DummyAtom previousAtom, ReadOnlyCollection<At
 
         internal RowAtom(SourceSpan source, IEnumerable<Atom> elements)
             : base(source) =>
-            this.Elements = elements.ToList().AsReadOnly();
+            this.Elements = elements.Where(x => x != null ).ToList().AsReadOnly();
```

Comment:
This shouldn't be necessary because `RowAtom` doesn't have any means to add `null` to its element list, and thus we shouldn't check for `null` here. The `elements` is really an enumerable of non-null atoms.

Reply:
I retract this statement: it looks necessary. We'll fix it with C# 8, then :)